background.js

```diff
@@ -161,8 +161,27 @@ const runJob = (job, retry = 0) => {
 }
 
 const init = async () => {
+
+  const jobsToStart = [];
+  for(let i = 2; i < process.argv.length; i++) {
+    jobsToStart.push(process.argv[i]);
+  }
+
+  if(jobsToStart.length > 0) {
+    console.log(`Will only start following jobs: ${jobsToStart.join(', ')}`);
+  }
+  else {
+    console.log(`Will start all jobs`);
+  }
+
   for(let job of jobs) {
     console.log(job);
+
+    if(jobsToStart.length > 0 && !jobsToStart.includes(job.file)) {
+      console.log(`Will not start ${job.file}`);
+      continue;
+    }
```

Comment:
this changes allow to run the background.js but for only a few jobs

examples: 
`node background.js ParseAave` will only run the server for aave
`node background.js ParseAave ParseAgave ParseMaker` will only run the server for these 3 parsers

If no argument is passed, then it run for all the jobs